tests/test_charge_point.py

```diff
@@ -318,7 +318,7 @@ def on_remote_stop_transaction(self, **kwargs):
     @on(Action.SetChargingProfile)
     def on_set_charging_profile(self, **kwargs):
         """Handle set charging profile request."""
-        return call_result.SetChargingProfilePayload(ChargingProfileStatus.accepted)
+        return call_result.SetChargingProfilePayload(ChargingProfileStatus.rejected)
```

Comment:
I don't think this has anything to do with the latency measurement?!

Reply:
I guess this is only for increasing test coverage.

Reply:
Correct,  it was just to get test coverage >90%